types/fundamental/misc/read.C

```diff
@@ -0,0 +1,57 @@
+#include <ROOT/REntry.hxx>
+#include <ROOT/RNTupleReader.hxx>
+
+using ROOT::Experimental::REntry;
+using ROOT::Experimental::RNTupleReader;
+
+#include <cstddef> // for std::byte
+#include <fstream>
+#include <ostream>
+#include <string>
+#include <string_view>
+
+template <typename T>
+static void PrintIntegerValue(const REntry &entry, std::string_view name,
+                              std::ostream &os, bool last = false) {
+  T value = *entry.GetPtr<T>(name);
```

Comment:
What happens here if the field type is incompatible with `T`?  (If it throws, I guess there is nothing to do) (It can happen if this script is used to read a mal-formed file).

Reply:
Yes, there will be an exception. For example, when trying to read `Char` as `int`, I get
```
terminate called after throwing an instance of 'ROOT::Experimental::RException'
  what():  type mismatch for field Char: char vs. std::int32_t
```
This should be sufficient (for the moment).

Reply:
Indeed good enough.